### PR TITLE
Fixing external URL

### PR DESCRIPTION
**Pull request by mbonell**
Makes the URL to be open in a new tab and show the "external" icon next to its name on the menu

### CODE REVIEW

File: docs/source/config.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
       "Federated schema checks": "/managed-federation/federated-schema-checks",
       "Deployment best practices": "/managed-federation/deployment",
       "Opt-in error reporting": "/managed-federation/error-reporting",
-      "Studio features": "https://www.apollographql.com/docs/studio/federated-graphs/"
+      "Studio features": "https://www.apollographql.com/docs/studio/federated-graphs"
```
Comment (Meschreiber) [inline docs/source/config.json:45]: It looks like https://www.apollographql.com/docs/studio/federated-graphs now redirects to https://www.apollographql.com/docs/graphos/graphs/federated-graphs/ so we might as well send folks directly there :) 
```suggestion
      "Studio features": "https://www.apollographql.com/docs/graphos/graphs/federated-graphs"
```

**Review by Meschreiber — COMMENTED** [review]
Thanks for this! I'm not seeing the external icon nor experiencing a new window opening currently though: https://www.loom.com/share/ade009351f86478f80e6705c7a89b2a0

Am I looking in the same place?

**Review by Meschreiber — APPROVED** [review]

**Comment by mbonell** [comment]
Thanks for the suggestion, I updated the url on a new commit.


> Thanks for this! I'm not seeing the external icon nor experiencing a new window opening currently though: https://www.loom.com/share/ade009351f86478f80e6705c7a89b2a0
> 
> Am I looking in the same place?

My understanding is that once the `/` is removed from the end of the URL, the icon and new experience will be applied. I didn't test it on sandbox so I cannot confirm it will 😟

**Comment by Meschreiber** [comment]
@peakematt , as you led the semgrep initiative, any chance you can help us figure out why this PR that just updates a docs link keeps failing the CI check? Is it because it's coming from a fork? TIA!

**Comment by mbonell** [comment]
> @peakematt , as you led the semgrep initiative, any chance you can help us figure out why this PR that just updates a docs link keeps failing the CI check? Is it because it's coming from a fork? TIA!

I forgot to create a new branch for this change :( not sure if that might be the issue 😞

**Review by dariuszkuc — APPROVED** [review]

**Pull request merged**